assets/src/scss/new-identity/style.scss

```diff
@@ -1,12 +1,17 @@
 @import "../base/tokens";
+@import "../base/variables";
+@import "../base/mixins";
 
 @mixin shared-link-styles {
+  --link--height: 26px;
+  --box-shadow--offset-y: calc(var(--link--height) * -1);
```

Comment:
Do we still need this variable? I don't see it reassigned anywhere 🤔

Reply:
It was part of the previous implementation,